### PR TITLE
app-shells/autojump: version bump

### PR DESCRIPTION
**Pull request by lukipuki**
Package-Manager: Portage-2.3.8, Repoman-2.3.3

### CODE REVIEW

File: app-shells/autojump/autojump-22.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline app-shells/autojump/autojump-22.5.1.ebuild]: Could you bump to EAPI version 6 whilst at it?

File: app-shells/autojump/autojump-22.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	doins bin/_j
+
+	python_foreach_impl python_domodule bin/autojump_argparse.py bin/autojump_data.py \
+		bin/autojump_match.py bin/autojump_utils.py
+	if use ipython; then
+		python_foreach_impl python_domodule tools/autojump_ipython.py
+	fi
+
+	doman docs/"${PN}.1"
+	dodoc README.md
```
Comment (monsieurp) [inline app-shells/autojump/autojump-22.5.1.ebuild]: Remove this line and call `einstalldocs` instead.

File: app-shells/autojump/autojump-22.5.1.ebuild (hunk 1 above)
Comment (lukipuki) [reply app-shells/autojump/autojump-22.5.1.ebuild]: Done. I've needed to add `eapply_user`.

File: app-shells/autojump/autojump-22.5.1.ebuild (hunk 2 above)
Comment (lukipuki) [reply app-shells/autojump/autojump-22.5.1.ebuild]: Done.

**Comment by lukipuki** [comment]
Ebuild updated, PTAL.

**Comment by lukipuki** [comment]
I've opened https://bugs.gentoo.org/636528 for the version bump.

**Pull request closed without merging**